### PR TITLE
A couple of small updates

### PR DESCRIPTION
**Pull request by SwooshyCueb**
This gets the plugin (almost) building again

### CODE REVIEW

File: data.world.cmake [hunk 1]
```diff
@@ -8,7 +8,7 @@ set(TARGET_NAME "${IRODS_TARGET_NAME_PREFIX}-${POLICY_NAME}")
 
 include(IrodsExternals)
 
-IRODS_MACRO_CHECK_DEPENDENCY_SET_FULLPATH_ADD_TO_IRODS_PACKAGE_DEPENDENCIES_LIST(ELASTICCLIENT elasticlient0.1.0-1)
+IRODS_MACRO_CHECK_DEPENDENCY_SET_FULLPATH_ADD_TO_IRODS_PACKAGE_DEPENDENCIES_LIST(ELASTICCLIENT elasticlientd68e30e3-0)
```
Comment (korydraughn) [inline data.world.cmake:11]: This is because the externals package references unreleased code?

**Review by korydraughn — COMMENTED** [review]
Changes are good.

File: data.world.cmake (hunk 1 above)
Comment (SwooshyCueb) [reply data.world.cmake:11]: Looks released to me https://packages.irods.org/apt/pool/focal/main/i/irods-externals-elasticlientd68e30e3-0/

Comment (korydraughn) [reply data.world.cmake:11]: Oh. This repo is just behind.

Comment (SwooshyCueb) [reply data.world.cmake:11]: More context for the change: d68e30e3 is the current version in the externals repo, and is the only version in the repo for Ubuntu 20.

Comment (korydraughn) [reply data.world.cmake:11]: Got it. Please mark this as resolved.

**Review by korydraughn — APPROVED** [review]
We've never had a release of this.

Add pounds when ready.

**Comment by korydraughn** [comment]
Consider creating a new issue for the keywords in this repo and linking to that since the one in irods/irods is already closed/linked to a previous release of iRODS.

**Comment by SwooshyCueb** [comment]
Done and done

**Review by alanking — APPROVED** [review]

**Comment by korydraughn** [comment]
Don't forget to mark the comment as resolved too.

**Pull request merged**